android/kotlin/v2/AmpliApp/app/src/main/java/com/amplitude/ampli/IngestionMetadata.kt

```diff
@@ -0,0 +1,29 @@
+package com.amplitude.ampli
+
+import com.amplitude.common.jvm.ConsoleLogger
+import org.json.JSONException
+import org.json.JSONObject
+
+class IngestionMetadata (sourceName: String, sourceVersion: String) {
+     var sourceName : String
+     var sourceVersion : String
+    init {
+        this.sourceName = sourceName
+        this.sourceVersion = sourceVersion
+    }
```

Comment:
I think this is equivalent to:
```suggestion
class IngestionMetadata (var sourceName: String, var sourceVersion: String) {
```